examples/jetsnack/gradle.properties

```diff
@@ -8,6 +8,3 @@ compose.version=1.6.10
 # TODO: enable configuration cache once we update on kotlin 2.0
 # see https://youtrack.jetbrains.com/issue/KT-64851 (k/wasm issue)
 org.gradle.configuration-cache=false
```

Comment:
It looks like it should be already solved

Reply:
fixed